docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Set_an_Observation.md

```diff
@@ -0,0 +1,123 @@
+# Set an Observation
+
+This section describes how to set an Observation in your Azure IoT Hub.
+
+Setting an Observation lets your devices know what value changes and for what particular resources they need to notify you about. An Observation can be set for any or all the components of the [data model](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings/): objects, objects instances, and resources. Whenever there is a change in values, a device will send a Notify message to Coiote DM, which in turn will transfer it to Azure IoT Hub.
+
+In this section, you learn how to:
+
+  * Set an Observation on objects and resources
+  * Define Observation attributes
+  * See value changes for the observed objects and resources
+
+## Prerequisites
+
+1. An active Azure IoT Hub with hub owner access permissions.
+2. A Coiote DM user account with permissions to use the integration extension.
+3. A device group created in Coiote DM.
+4. [A configured extension between Coiote DM and Azure Iot Hub](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Configuring_Azure_IoT_Hub_integration_extension/).
+5. [A configured integration template in Coiote DM](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates/).
+5. [A connected device](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Device_operations/Overview/).
+
+## Set an Observation and define its attributes
+
+Let’s set two Observations: one on the whole object with ID **1/1 (1 LwM2M Server, 1 Lifetime)** and on the resource with ID **3/0/13** (3 Device, 13 Current time). Both of them are Properties.
+
+In Azure IoT Hub, value changes for both Telemetry and Property are stored in **Device twin**, because Observation is a Property in itself and it does not keep track of historical changes in values. To set an Observation, do the following:
```

Comment:
I'd put this link under 'Device twin' https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings/#lwm2m-readable-and-writable-resources

Reply:
thanks, will add the link